.github/workflows/release.yaml

```diff
@@ -32,7 +90,6 @@ jobs:
       base64-subjects: ${{ needs.build.outputs.artifacts-sha256 }}
 
   release:
-    if: ${{ github.ref == 'refs/heads/main' && !startsWith(github.event.commits[0].message, 'bump:') }}
```

Comment:
Removing this guard is ok because `build` may not run, therefore `release` doesn’t run?

Reply:
> Removing this guard is ok because `build` may not run, therefore `release` doesn’t run?

Yes, that's correct. Here is the dependency chain which saves us from repeating the guard:
```
check -> bump (guarded) -> build -> provenance -> release
```